config/cypress.yml

```diff
@@ -2,7 +2,7 @@
 # upgrades failing to complete successfully.
 
 # These settings are kept in the constants file with the assumption that the end user should not be overriding them.
-bundle_file_path: 'temp/bundles'
+bundle_file_path: 'tmp/bundles'
```

Comment:
Nice, always wondered why there was a temp and tmp directory but I never dug into why. As long as the bundle files are never used after import this should be fine, we do run a `tmp:cache:clear` on upgrades, so that is the only potential problem I can think of.

Reply:
AFAIK the bundle files are never used after import, so we should be safe here.